apps/survey/contracts/Survey.sol

```diff
@@ -41,9 +41,9 @@ contract Survey is AragonApp {
     struct SurveyStruct {
         address creator;
         uint64 startDate;
-        uint256 options;
-        uint256 snapshotBlock;
-        uint256 minParticipationPct;
+        uint64 snapshotBlock;
+        uint64 minParticipationPct;
+        uint64 options;
```

Comment:
I think we should keep `options` to be `uint256` as `optionPower` still uses `uint256` for its mapping. In general, I think any "length"-type parameter should be kept to `uint256`.

Reply:
Fair point, I'll revert.